superset-frontend/src/explore/components/controls/ConditionalFormattingControl/FormattingPopoverContent.tsx

```diff
@@ -18,16 +18,16 @@
  */
 import React from 'react';
 import { styled, SupersetTheme, t, useTheme } from '@superset-ui/core';
+import {
+  Comparator,
+  MultipleValueComparators,
+} from '@superset-ui/chart-controls';
```

Comment:
Double checking... is this being pulled from the right place?

Reply:
looks like the import changes... wasn't sure that's intended.

Reply:
Yes - both `Comparator` and `MultipleValueComparators` were duplicated, and existed both here and in `superset-ui/core`. In this PR I removed the one here and replaced it with references to `superset-ui/core`.